packages/lambda-tiler/src/routes/sprites.ts

```diff
@@ -16,6 +16,7 @@ Extensions.set('.png', 'image/png');
 Extensions.set('.json', 'application/json');
 
 export async function spriteGet(req: LambdaHttpRequest<SpriteGet>): Promise<LambdaHttpResponse> {
+  //TODO: Replace this with cb_.assets
```

Comment:
Are we still going to set assets location here or in `router.hook` and give a assetprovider for all of them.